app/lib/features/chat/dialogs/create_chat_sheet.dart

```diff
@@ -44,8 +45,8 @@ class _CreateChatSheetConsumerState extends ConsumerState<CreateChatSheet> {
     if (widget.initialSelectedSpaceId != null) {
       isSpaceRoom = true;
       Future(() {
```

Comment:
Why do we want ```Future``` here? Firstly the internal call isn't async and even if there should be, we can't directly  do async operations in widget lifecycle methods. Secondly, we are changing provider state here which could lead to exception while widget's own state is volatile. Please use ```WidgetInstanceBinding.instance.addPostFrameCallBack()``` in place of ```Future``` to ensure we doing provider state changes after widget is properly initialized.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/855/commits/81b669d08592535b7e6344fdaea9d2ecaa89c08d